src/LayerEditor/ui/tools/selection.py

```diff
@@ -54,3 +54,7 @@ def max_selected_dim(self):
             return max([ g_item.dim for g_item in self._selected])
         else:
             return 0
+
+    def deselect_item_if_exists(self, g_item):
```

Comment:
Where is it used?

Reply:
Apparently it is artifact from the original `selection` from Genie. I removed it. Done